packages/app/features/Send/TransactionConfirmation/components/BigToken.tsx

```diff
@@ -7,14 +7,18 @@ import { transactionContext } from '../../../../state/transaction';
 export const BigToken = () => {
 	const { token, amount } = useSnapshot(transactionContext);
 
-	const iconUri = { uri: token?.metadata?.imageUri };
+	const iconUri = {
+		uri: token?.metadata?.imageUri ?? '/img/send-token/unknown-token.jpeg',
+	};
 
 	return (
 		<View style={styles.container}>
 			<Image style={styles.tokenIcon} source={iconUri} />
 			<View style={styles.amountContainer}>
 				<Text style={styles.amountText}>{amount}</Text>
-				<Text style={styles.symbolText}>{token?.metadata?.symbol}</Text>
+				<Text style={styles.symbolText}>
+					{token?.metadata?.symbol ?? 'Unknown'}
```

Comment:
check again

Reply:
i handle this case for some tokens which are mint unofficially so that we cannot fetch their metadata on solana